StackAsLinkedList.java

```diff
@@ -0,0 +1,84 @@
+// Time Complexity : O(n)
+// Space Complexity : O(N)
+// Did this code successfully run on Leetcode :
+// Any problem you faced while coding this : N/A
+class StackNode {
+    int data;
+    //Creating a doubly linked list
+    StackNode next;
+    StackNode prev;
+    //Implementing it through doubly linkedList.
+
+    //Creating constructors
+    StackNode(){
+
+    }
+    StackNode(int data) {
+        //Constructor here
+        this.data = data;
+        this.next = null;
+        this.prev = null;
+    }
+    StackNode(int data, StackNode next) {
+        //Constructor here
+        this.data = data;
+        this.next = next;
+        this.prev = null;
+    }
+    StackNode(int data, StackNode next,StackNode prev) {
+        //Constructor here
+        this.data = data;
+        this.next = next;
+        this.prev = prev;
+    }
+}
+public class StackAsLinkedList {
+
+    StackNode root;
+
+    public boolean isEmpty() {
+        //Write your code here for the condition if stack is empty.
+        if(root == null){ // comparing whether the root is null or not
+            return true;
+        }
+        return false;
+    }
+
+    public void push(int data) {

```

Comment:
Actually you have to insert a node at beginning of the linked list as you're implementing stack.

Reply:
I put a check in push() method, if there is no node, then it will create a node(head) and then insert the value in the head node. I believe it will work the same.